packages/paste-website/src/components/tokens-example/index.tsx

```diff
@@ -17,10 +17,15 @@ export const ColorBox: React.FC<BackgroundColor> = ({backgroundColor}) => {
   return <Absolute backgroundColor={backgroundColor} padding="space50" preset="fill" />;
 };
 
-type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
-export const BorderBox: React.FC<BorderBoxProps> = ({borderColor, borderWidth}) => {
+type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth' | 'height'>;
+export const BorderBox: React.FC<BorderBoxProps> = ({borderColor, borderWidth, height}) => {
   return (
-    <Box borderStyle="solid none none none" borderColor={borderColor} borderWidth={borderWidth || 'borderWidth20'} />
+    <Box
+      borderStyle="solid none none none"
+      borderColor={borderColor}
+      borderWidth={borderWidth || 'borderWidth20'}
+      height={height || undefined}
```

Comment:
We should be able to just do `height={height}` here.

Reply:
Done